WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostPagerActivity.java

```diff
@@ -708,6 +705,7 @@ public void run() {
                         // let the user know they can swipe between posts
                         if (adapter.getCount() > 1 && !AppPrefs.isReaderSwipeToNavigateShown()) {
```

Comment:
This is not directly related to this issue, but when I open any post other then first, `adapter.getCount()` returns 1 and the toast is not shown 🤔

Reply:
`NotificationsDetailActivity` works as expected.

Reply:
@khaykov Sorry, I only just now realized you commented here! I'm not able to reproduce the issue with `getCount()` returning 1, and looking at the code I don't see how it's possible unless there really is only one item. Could you tell me the steps to repro that?

Reply:
Huh. It works as expected now :) Sorry about that! I blame 👻